tools/funcslower.py

```diff
@@ -142,6 +165,28 @@
     data.start_ns = entryp->start_ns;
     data.duration_ns = delta_ns;
     data.retval = PT_REGS_RC(ctx);
+
+#ifdef USER_STACKS
+    data.user_stack_id = stacks.get_stackid(ctx, BPF_F_REUSE_STACKID | BPF_F_USER_STACK);
```

Comment:
if user specified pid based filtering, `BPF_F_REUSE_STACKID` can be used. Otherwise, it may cause issues as reusing stack id may cause the stack id associating with a different pid.